### ISSUE TITLE
task: Improve documentation around INFRAHUB_PROXY & INFRAHUB_PROXY_MOUNTS

### ISSUE THREAD

**Issue by BeArchiTek** [opsmill/infrahub#3991]
### Component

Documentation & Examples

### Task Description

There is no information regarding the behavior and usage of the INFRAHUB_PROXY by the SDK in the documentation

**Comment by wvandeun** [comment]
There is this, but I agree this is the absolute minimum:
https://docs.infrahub.app/python-sdk/reference/config#proxy

**Comment by BeArchiTek** [comment]
Yeah i saw it and that's kinda how I found the 2 env variables, but had to read the code to understand what it was doing :)
